### PR TITLE
TASK-52353:Improve the UI of topbar in perk store for mobile views

### PR DESCRIPTION
**Pull request by GouadriaHanen**

### CODE REVIEW

File: perk-store-webapps/src/main/webapp/css/main.less [hunk 1]
```diff
@@ -1038,27 +1044,34 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
...
-            margin-top: 15px;
+            max-width: 67%;
+            margin-top: 7px;
           }
           .filter_menu {
-            width: fit-content;
-            text-align: end;
-            margin-top: 10px;
+            max-width: 30%;
+            margin-right: 16px;
```
Comment (ahamdi) [inline perk-store-webapps/src/main/webapp/css/main.less]: Should we check this in RTL languages configuration ?

File: perk-store-webapps/src/main/webapp/css/main.less [hunk 2]
```diff
@@ -1038,27 +1044,34 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
       .toolbarProduct {
         .addProduct {
           width: fit-content;
+          margin-bottom: 12px;
         }
         .filter {
           width: 100% ;
-          margin: 0 -2%;
+          margin-right: -4%;
```
Comment (ahamdi) [inline perk-store-webapps/src/main/webapp/css/main.less]: Should we check this in RTL languages configuration ?

File: perk-store-webapps/src/main/webapp/css/main.less [hunk 3]
```diff
@@ -1176,9 +1196,41 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
         }
       }
     }
+    .product-list {
```
Comment (ahamdi) [inline perk-store-webapps/src/main/webapp/css/main.less:1201]: Could we avoid duplicating those styles ?

Comment (GouadriaHanen) [reply perk-store-webapps/src/main/webapp/css/main.less:1201]: for each media size we need a different style

**Review by ahamdi — APPROVED** [review]

**Pull request merged**